appveyor.approval.yml

```diff
@@ -0,0 +1,18 @@
+version: '{build}'
+
+environment:
+    PATH: 'C:\Ruby200\bin;%PATH%'
+
+install:
+    - cmd: bundle install
+
+build_script:
+    - cmd: build.netstd.cmd approve

```

Comment:
Perhaps we should bite the bullet and make `approve` part of the regular build. It may cause *a little* friction for contributors, but we can always send a superseding PR if the burden of updating the approved API is too much for them.